runtime/engine/src/main/java/io/aklivity/zilla/runtime/engine/binding/BindingContext.java

```diff
@@ -29,4 +31,10 @@ default void detach(
         BindingConfig binding)
     {
     }
+
+    default void accept(
+        long bindingId,
+        SocketChannel channel)
+    {
+    }
```

Comment:
This is not needed on generic `BindingContext`, only on `TcpBindingContext`, right?